02-tests/02-tests.md

```diff
@@ -1,8 +1,8 @@
-## Goal: Perform test applications on various platforms and check that they run as expected
+Summary
+=======
 
-
-The `./compile_and_test_for_board.py` script can be used to run all compilation
```

Comment:
You refer to the script in a newly introduced sentence in #119. Do you want to keep this line in?

Reply:
The sentence in https://github.com/RIOT-OS/Release-Specs/pull/119 is correct - what happens here depends I guess on which order these PRs get merged? (Does that answer your question?)